### ISSUE TITLE
[BP] Some sequences do not trigger the ragdoll

### ISSUE THREAD

**Issue by di57inct** [hzqst/MetaHookSv#167]
I even tried setting it to frame 0 but got the same result.

https://user-images.githubusercontent.com/3608544/175128671-f4e365fc-69a3-433b-9abb-f224f02b792f.mp4

**Comment by hzqst** [comment]
![image](https://user-images.githubusercontent.com/12287588/175199011-12f92827-9264-49fd-b9bb-9660e098574a.png)

Currently only sequences with any of the following Activity are considered as DeathAnim :
 ```
	if (
		pseqdesc->activity == ACT_DIESIMPLE ||
		pseqdesc->activity == ACT_DIEBACKWARD ||
		pseqdesc->activity == ACT_DIEFORWARD ||
		pseqdesc->activity == ACT_DIEVIOLENT ||
		pseqdesc->activity == ACT_DIEVIOLENT ||
		pseqdesc->activity == ACT_DIE_HEADSHOT ||
		pseqdesc->activity == ACT_DIE_CHESTSHOT ||
		pseqdesc->activity == ACT_DIE_GUTSHOT ||
		pseqdesc->activity == ACT_DIE_BACKSHOT
		)
	{
		return 1;
	}
```

**Comment by hzqst** [comment]
Should have been fixed in commit 5c32f12e67411deac46266f55a09932865c9f9fb, now any sequence listed in [DeathAnim] are treated as death no matter what Activity it is.

**Comment by di57inct** [comment]
Looks like it's fixed. Thanks ^^
